refactor(interchange): Simplify order books implementation (#729)
cosmos/sdk-tutorials: interchain-exchange/tutorial/03-orderbook.md

In this chapter you implement the code for the order book, for buy orders and sell orders.

> In this chapter you will create a `order_book.go` file with the implementation for the order book. 

Reviewer: ```suggestion
In this chapter, you create an `order_book.go` file with the implementation for the order book. 
```
